web/src/compose_validate.ts

```diff
@@ -867,7 +881,7 @@ export function validate(scheduling_message: boolean): boolean {
     if (compose_state.get_message_type() === "private") {
         return validate_private_message();
     }
```

Comment:
This condition is repeated. Also, I like using `validate_private_message` inside the if condition. Let's use it above too for *both* stream and private if conditions.